docs/content/users/quickstart.md

```diff
@@ -587,6 +614,30 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch
     ```
 
+## Symfony
+
+=== "Composer"
+
+    ```bash
+    mkdir my-symfony && cd my-symfony
+    ddev config --docroot=public
+    ddev composer create symfony/skeleton:"7.0.*"
+    ddev composer require webapp
+    # When it asks if you want to include docker configuration, say "no" with "x"
+    ddev launch
+    ```
+
+=== "Git Clone"
+
+    ```bash
+    git clone <your-symfony-repo>
+    cd <your-symfony-repo>
+    ddev config --docroot=public --php-version=8.3
+    ddev start
+    ddev composer install
+    ddev launch
+    ```
```

Comment:
I'd suggest a couple of links with more information until we can have a symfony project-type 

```suggestion
    ddev launch
    ```
As Symfony projects can be so flexible, you might need different customizations for your project.

You might want to start by editing your `.env` file for setting the DB connection string if you require one. See the [official docs](https://symfony.com/doc/current/doctrine.html#configuring-the-database) for more information.
If you require your current database engine version, you can use `ddev -s db exec --raw mysql -udb -pdb db -e 'SELECT VERSION();'`

Read more about customizing the environment with different [Config Options](https://ddev.readthedocs.io/en/latest/users/configuration/config/) and [Defining Additional Services with Docker Compose](https://ddev.readthedocs.io/en/latest/users/extend/custom-compose-files/).    
    
```

Reply:
I guess you don't know about `ddev mysql`. You'll like it! Please edit to use it. 

But... I don't understand what "if you require your current database engine version" means. Wouldn't they just `ddev describe` ?

The links should be relative links (in docs) instead of explicit links. Can you edit?

Reply:
Updated those.

The version suggestion is because by default doctrine uses a DSN, and looks like it's a bit picky about those with mariadb/mysql according to the official Doctrine DBAL docs, so you might want to include the version in the DSN connection string.

https://www.doctrine-project.org/projects/doctrine-dbal/en/4.0/reference/configuration.html#automatic-platform-version-detection

Reply:
Just `ddev mysql  -e 'SELECT VERSION();'` :)

But not sure how it's better than `ddev describe`

Reply:
It gives the exact major.minor.patch version. Honestly, don't know how relevant that is, but most of examples in those docs specify at that level.

Reply:
I added a commit with your suggestion and minor changes.